src/index.js

```diff
@@ -82,7 +94,7 @@ const mondayTemperature = monday.temperature
 const mondayCloudCoverage = monday.cloudCoverage
 const mondayWind = monday.wind
 
-
+// I dont understand this 
```

Comment:
It's ok 

It's to do with destructuring which was in the study materials but we didn't have time to cover in the session. 
I will show it in one of our sessions. 

``` javascript
const { wind, cloudCoverage, temperature } = monday
```